### ISSUE TITLE
Head on wall bug

### ISSUE THREAD

**Issue by DramaLvl1** [HimmelKreis4865/BetterSkulls#8]
**Describe the bug**
The head entity didn't got removed

**To Reproduce**
1. Give a head
2. Place the head on a wall
3. Place water next to the head

**Expected behavior**
The head entity should got removed by the flowed water.

**Screenshots**

**Desktop:**
 - OS: Windows 10 Home 20H2
 - PHP Version 7.4
 - PocketMine-MP Version: 3.19.0

**Comment by HimmelKreis4865** [comment]
yeah, thanks for the report, this is actually untested but I guess it could be fixed tomorrow

**Comment by galaxytwenty** [comment]
same with TNT xD
